extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -598,8 +614,8 @@ export class AmpSlideScroll extends BaseSlides {
    * @private
    */
   hideRestOfTheSlides_(indexArr) {
-    const noOfSlides_ = this.noOfSlides_;
-    for (let i = 0; i < noOfSlides_; i++) {
+    const totalNumSlides = this.slides_.length;
```

Comment:
If `this.noOfSlides_` is lower than the true number of slides, as it will be if `[slide-count]` is mutated to a lower number, then this method will fail to hide the slides between this.noOfSlides_ and this.slides_.length